packages/angular-workspace/nimble-angular/CONTRIBUTING.md

```diff
@@ -47,4 +47,6 @@ The Angular directives need code coverage via unit tests.
 
 ## Updating Angular
 
-When updating the version of Angular used by the package, be sure to also update the versions of forked Angular files in `src/thirdparty/directives`. See the README in the thirdparty folder for details.
\ No newline at end of file
+When updating the version of Angular used by the package:
+- Align with [supported versions of associated libraries](https://angular.dev/reference/versions) (e.g. `typescript` and `rxjs`).
```

Comment:
We should add a step here that says something like, "Because Angular tends to have many associated dependencies that update, first make sure that the nimble `package-lock.json` is up-to-date on main (i.e. very recently rebuilt) and then perform the Angular update by rebuilding the `package-lock.json` for the new Angular version PR"